### PR TITLE
remove spaces in license

### PR DESCRIPTION
**Pull request by Alessandro-Barbieri**

### CODE REVIEW

File: ebuild_generator/ebuild.py [hunk 1]
```diff
@@ -176,7 +176,7 @@ def get_ebuild_text(self, distributor, license_text, die_msg=None):
                 ret += ')"\n'
             else:
                 ret += "LICENSE=\""
-                ret += get_license(self.upstream_license) + "\"\n\n"
+                ret += get_license(self.upstream_license.replace(' ', '')) + "\"\n\n"
```
Comment (allenh1) [inline ebuild_generator/ebuild.py]: I'd prefer the get_license function have this kind of line for the default return case (the one selected when none of the regexes match).

**Comment by allenh1** [comment]
This brings to light the need for better licenses, though. There are a lot of silly ones in the package.xml files like "ToDo" that are obviously not real licenses. Any ideas as to how this should be dealt with?

File: ebuild_generator/ebuild.py (hunk 1 above)
Comment (Alessandro-Barbieri) [inline ebuild_generator/ebuild.py]: where I should remove spaces then?

Comment (allenh1) [inline ebuild_generator/ebuild.py]: https://github.com/allenh1/ebuild-generator/blob/2709852666fdec9ce54fd904c559fea5e7e83795/ebuild_generator/ebuild.py#L79

**Comment by allenh1** [comment]
Multiple licenses apply. It's a legal requirement of the license to distribute all licenses with it, so it's entirely possible to have a single file under multiple licenses, then the rest of a repo under a different one. It's kinda strange, but so it is.

**Comment by Alessandro-Barbieri** [comment]
Ok then I'll remove `||`

**Comment by allenh1** [comment]
Sounds like a good idea to me.

**Pull request merged**